lblod/ember-rdfa-editor-lblod-plugins
# GN-4937: update snippet-preview component to be collapsible

### Overview
This PR updates the `snippet-preview` component of the snippet-plugin to contain a collapsible (by default not shown) preview of the snippet.
Specifically, this PR includes the following:
- `snippet-preview` component: move to `.gts` file format
- `snippet-preview` component: update css
- `snippet-preview` component: implement collapsible snippet preview

##### connected issues and PRs:
[GN-4937](https://binnenland.atlassian.net/browse/GN-4937?atlOrigin=eyJpIjoiOTg4ZDg4YmU5ZDViNDJhN2E4ZTM2NjBiZjg4MjNjMzUiLCJwIjoiaiJ9)

### How to test/reproduce
- Start the dummy app
- Insert a snippet placeholder with some snippet-lists configured
- Open the 'Insert snippet' modal
- Notice that for each of the snippets in the list, you can preview the snippet
- If a snippet does not have any content, an italic message is shown (should we do this with a warning?)

### Challenges/uncertainties
- Should we show a warning instead of a simple message when a snippet is empty?
- I did not opt for shortening the snippet preview, as this did not seem necessary. Correct me if I'm wrong.



### Checks PR readiness
